internal/bgp/frr/testdata/TestSingleAdvertisementChange.golden

```diff
@@ -30,6 +32,14 @@ router bgp 100
   address-family ipv4 unicast
     neighbor 10.2.2.254 activate
     neighbor 10.2.2.254 route-map 10.2.2.254-in in
-    network 172.16.1.11/24
     neighbor 10.2.2.254 route-map 10.2.2.254-out out
   exit-address-family
+  address-family ipv6 unicast
+    neighbor 10.2.2.254 activate
+    neighbor 10.2.2.254 route-map 10.2.2.254-in in
+    neighbor 10.2.2.254 route-map 10.2.2.254-out out
+  exit-address-family
```

Comment:
From looking at the `TestSingleAdvertisementChange` test It shouldn't have an ipv6 family here I think

Reply:
This the only effective change introduced by this PR. We always add the peers to both IPv4 and IPv6 families instead of adding to both if there are no advertisements, and adding them only to the family where we have advertisements in case at least one advertisement is present (which causes the route flapping), as explained in https://github.com/metallb/metallb/issues/1715

Reply:
ok, now I get it. thanks,